arangod/RestServer/LockfileFeature.cpp

```diff
@@ -79,6 +79,13 @@ void LockfileFeature::start() {
       << _lockFilename << "': " << TRI_errno_string(res);
     FATAL_ERROR_EXIT_CODE(TRI_EXIT_COULD_NOT_LOCK);
   }
+
+  auto cleanup = std::make_unique<CleanupFunctions::CleanupFunction>(
+  [&] (int code, void* data) {
+    LOG_TOPIC(ERR, arangodb::Logger::FIXME) << "Removing LOCK file";
```

Comment:
please remove this line, as it is no error!

Reply:
Agreed